docs/changes/io.votable/17570.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed a bug where VOTable writes out unit in CDS format instead of VOUnit.
```

Comment:
The change log entry makes it sound worse than it really is. VOUnit was designed to be backwards-compatible with CDS, so in most cases the tables that were written are still valid. This should also be apparent from the fact that you didn't have to modify any of the existing tests that involve writing a VOTable. The users who didn't get any unexpected errors with `astropy` v7.0 don't have too much to worry about.

Reply:
Please suggest desired text. Thanks!